app/repositories/submission_repository.rb

```diff
@@ -11,14 +11,14 @@ def to_rate
     to_rate_scope.to_a
   end
 
-  def next_to_rate(current_created_at)
-    to_rate_scope.where('submissions.created_at > ?', current_created_at).order('created_at ASC')
-      .first || to_rate_scope.first
+  def next(current_created_at)
+    not_rejected.where('submissions.created_at > ?', current_created_at).order('created_at ASC')
+      .first || not_rejected.first
```

Comment:
> this || is here for not passing current_created_at case?

looking at this line it seems to me that they wanted to make "a circle" here, so after the last submission comes the first one again